### PR TITLE
{WIP}Test Appveyor Multilang

### PR DESCRIPTION
**Pull request by glennsarti**
DO NOT MERGE

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,16 +1,54 @@
...
+        Write-Output "Restarting worker in 5 seconds...";
+        Start-Sleep -Seconds 5
+        Restart-Computer -Force -Confirm:$false;
+      } else {
+        Write-Output "Locale is already set to $currentLocale";
+      };
+
+test_script:
+  # The REMs are needed to help AppVeyor NOT to lose console messages after a reboot (locale change)
+  # Typcially the first 2 or 3 are dropped.  Don't know why. Just Coz.
```
Comment (kylog) [inline appveyor.yml]: How does software even work?

Comment (glennsarti) [inline appveyor.yml]: It's a Windows thing :-)

**Comment by joshcooper** [comment]
@glennsarti ok to close this since we're already testing JP in Appveyor?

**Comment by glennsarti** [comment]
Whoops.  Yes.

**Pull request closed without merging**